Test/Microsoft.ApplicationInsights.Test/Shared/Extensibility/TelemetryConfigurationFactoryTest.cs

```diff
@@ -275,14 +275,15 @@ public void LoadInstanceSetsInstancePropertiesOfTimeSpanTypeFromChildElementValu
         }
 
         [TestMethod]
+        [ExpectedExceptionWithMessage(typeof(ArgumentException), "Failed to parse configuration value. Property: 'TimeSpanProperty' Reason: String was not recognized as a valid TimeSpan.")]
         public void LoadInstanceSetsInstancePropertiesOfTimeSpanTypeFromChildElementValuesOfDefinitionWithInvalidFormatThrowsException()
```

Comment:
can you add a unit test like the example you shared in the gh issue?  showing new TelemetryClient() throwing exception as expected.

Reply:
I would be happy to, but I'm not sure how exactly.

Do we have any example tests like that?

In a real program, `new TelemetryClient` will attempt to load a config file. I would need to mock something to load a config string

Reply:
new TelemetryClient(TelemetryConfig.FromFile(badfile))